### PR TITLE
Glitchexperiment

### PR DESCRIPTION
**Pull request by manujawicky**

### CODE REVIEW

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Hey there, I have a few changed that must be made in order for me to merge this. I will highlight them.

File: animations/glitchexperiment.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
+body {
+  background: black;
+  font-family: "Oswald", sans-serif;
+  font-style: italic;
+}
```
Comment (NiallEccles) [inline animations/glitchexperiment.css:5]: Could you move these styles from the body to the .container below?  Also you would need to add in this outside of a selector `@import url('https://fonts.googleapis.com/css2?family=Oswald&display=swap'); `. Currently the only way to view your selected font is to have that font installed.

**Comment by manujawicky** [comment]
It's glad you review my request, Can you please tell me what parts should I change.

**Review by NiallEccles — CHANGES_REQUESTED** [review]

**Comment by manujawicky** [comment]
I have do the changes as you said. I think now you can see the font style.

**Pull request merged**